From 98e6d3ea7aa540d68edb5e3773cd7d85b574fc46 Mon Sep 17 00:00:00 2001
From: Sergey Larin <cerg2010cerg2010@mail.ru>
Date: Thu, 20 Jun 2019 20:24:45 +0300
Subject: [PATCH] ARM: dts: tegra20-glide: Add binding for FSA9480

FSA9480 is a USB cable detector and switch. This board uses this
to provide USB, USB-OTG, MHL and UART functionality over one microUSB port.

Signed-off-by: Sergey Larin <cerg2010cerg2010@mail.ru>
---
 arch/arm/boot/dts/tegra20-glide.dts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/tegra20-glide.dts b/arch/arm/boot/dts/tegra20-glide.dts
index b8f3916ec35b..fc4aca8a1c47 100644
--- a/arch/arm/boot/dts/tegra20-glide.dts
+++ b/arch/arm/boot/dts/tegra20-glide.dts
@@ -941,13 +941,13 @@
 		i2c-gpio,delay-us = <1>;
 		#address-cells = <1>;
 		#size-cells = <0>;
-		interrupt-parent = <&gpio>;
-		interrupts = <TEGRA_GPIO(L, 1) IRQ_TYPE_LEVEL_HIGH>;
 
-		/* Not supported - no DT bindings */
-		/* fsa9480@25 {
+		fsa9480@25 {
+			compatible = "fcs,fsa9480";
 			reg = <0x25>;
-		}; */
+			interrupt-parent = <&gpio>;
+			interrupts = <TEGRA_GPIO(L, 1) IRQ_TYPE_LEVEL_HIGH>;
+		};
 	};
 
 	hdmi_ddc: i2c@13 {
-- 
2.22.0

